.travis.yml

```diff
@@ -1,20 +1,8 @@
-language: android
```

Comment:
I think you need to leave all the version updates and travis changes for this to pass on travisci.